kite-data/kite-data-core/pom.xml

```diff
@@ -271,5 +271,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
```

Comment:
Ah, I see, the difference is the `repositoryId` and `url`.

So next question: can we control that with a property instead? If we can, and we merge the two profiles, that means we would not put releases in the snapshot repository (which sounds good to me), and since this [line](https://github.com/scalingdata/cdk/pull/23/files#diff-1441627d1cf34e12817f363bf0b54effR50) sets both profiles at once and the maven docs linked to in another comment say the executions get merged since they have the same ID, then I'm guessing we aren't deploying to both the snapshot and release repos anyway.

The only thing I think we'd miss out on by merging the profiles is the `hdp2-release` profile would now get `install-hdp-classifier` execution, whereas right now it doesn't have that.